graylog2-web-interface/src/stores/inputs/InputsStore.ts

```diff
@@ -27,7 +27,7 @@ type InputsActionsType = {
   list: () => Promise<{ inputs: Array<Input>; total: number }>;
   get: (id: string) => Promise<Input>;
   getOptional: (id: string, showError: boolean) => Promise<Input>;
-  create: (input: ConfiguredInput) => Promise<void>;
+  create: (input: ConfiguredInput) => Promise<{ id: string }>;
```

Comment:
Do we also need to change the **create** fetch promise return, to make sure it is not a void and it is `returning the id`?

Reply:
The backend return the id on the create fetch Promise since we are returning the promise we can be sure that if it resolve it will have the id